src/ray/common/ray_config_def.h

```diff
@@ -88,6 +88,16 @@ RAY_CONFIG(uint64_t, memory_monitor_interval_ms, 0)
 /// eligble for garbage colletion.
 RAY_CONFIG(uint64_t, task_failure_entry_ttl_ms, 15 * 60 * 1000)
 
+/// The number of retries for the task or actor when
+/// it fails due to the process being killed when the memory is running low on the node.
+/// The process killing is done by memory monitor, which is enabled via
+/// memory_monitor_interval_ms. If the task or actor is not retriable then this value is
+/// ignored. This retry counter is only used when the process is killed due to memory, and
```

Comment:
> If the task or actor is not retriable then this value is ignored. 

Seems we don't have this logic?

Reply:
Should be in task_manager.cc:

int32_t max_oom_retries =
      (max_retries != 0) ? RayConfig::instance().task_oom_retries() : 0;